build_tools/cmake/iree_benchmark_suite.cmake

```diff
@@ -214,14 +214,14 @@ function(iree_benchmark_suite)
 
       # Register the target once and share across all benchmarks having the same
       # MLIR source and translation flags.
-      set(
-        _TRANSLATION_TARGET_NAME
-        "${PACKAGE_NAME}_iree-generate-benchmark-artifact-${_MODULE_SOURCE_BASENAME_WITH_HASH}"
+      set(_TRANSLATION_NAME
+        "iree-generate-benchmark-artifact-${_MODULE_SOURCE_BASENAME_WITH_HASH}"
       )
+      set(_TRANSLATION_TARGET_NAME "${_PACKAGE_NAME}_${_TRANSLATION_NAME}")
```

Comment:
So we still have to know the details of how iree_bytecode_module works to some degree. That's a bit unfortunate, though at least it's standardized.